app/javascript/utilities/moderation.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Determines whether or not a member is on the moderation page.
+ *
+ * @param {String} [path=top.location.pathname] The path to check.
+ *
+ * @returns {Boolean} True if the path is the moderation page, false otherwise.
+ **/
+export function isModerationPage(path = top.location.pathname) {
+  return path.endsWith('/mod') || path.endsWith('/mod/');
```

Comment:
This is the fix. Previously, the check was `path.includes('/mod')`. The issue with this is it will think a post with a URL like this, `/some_user/mod-not-the-moderation-page`, is the moderation page. `endsWith` is what we really want.